project/AppDependencies.scala

```diff
@@ -4,26 +4,27 @@ import sbt._
 
 object AppDependencies {
 
-  private val bootstrapPlayVersion = "7.23.0"
+  private val bootstrapPlayVersion = "8.1.0"
 
   val compile: Seq[ModuleID] = List(
     ws,
-    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28" % bootstrapPlayVersion,
+    "uk.gov.hmrc"                  %% "bootstrap-backend-play-30" % bootstrapPlayVersion,
     "org.typelevel"                %% "cats-core"                 % "2.9.0",
     "com.chuusai"                  %% "shapeless"                 % "2.4.0-M1",
     "com.neovisionaries"            % "nv-i18n"                   % "1.29",
-    "com.fasterxml.jackson.module" %% "jackson-module-scala"      % "2.14.2"
+    "com.fasterxml.jackson.module" %% "jackson-module-scala"      % "2.15.2",
+    "joda-time"                     % "joda-time"                 % "2.10.12"
```

Comment:
In the 'Play 3.0 Migration Guide' it says 'If you are still using Joda time, please replace this with Java time.'

Reply:
Replaced joda time classes with java time classes.